rfd/0036/projects/manifest.md

```diff
@@ -12,7 +12,11 @@
 
 The project manifest defines all the services of a project, as well as other details that may be added in time. Each project has a single active manifest that defines the expected state of the entire project.
 
+The `version` and `tags` fields are optional, and are present to help give the user meaningful output from commands like `triton project versions` or `triton project ls`, providing human-readable information in addition to sha1sums, and the ability to filter. See []
+
 ```yaml
+version: 1
```

Comment:
Yes, with a sha1sum. Not tied to this at all, but thought we may want to provide a way for users to provide version numbers meaningful to them.  After thinking through tagging (separate issue), I arrived at this. See examples I added for `triton project ls`, for example, on how this could provide more meaningful output.